### ISSUE TITLE
[Feature Request] Support for IVCS Bones possible?

### ISSUE THREAD

**Issue by StoneColb** [XIV-Tools/CustomizePlus#50]
Unsure of the core programming for the plugin, but if it pulls from the same data that Anamnesis does to adjust bone scales, would it be possible to add support for adjusting IVCS bones as well, assuming the user actually has IVCS?

The primary use for this I imagine would be allowing you to scale your character's butt without also scaling up the hip bones. Or better chest scaling.

I suppose the major concern would be what could happen if someone tried to adjust IVCS bones without them being present. Could be fine, could crash the game.

**Comment by StoiaCode** [comment]
If you import a scale from ana with IVCS bones I think it should already be possible? Haven't checked, but am decently sure.

**Comment by seirralanzce** [comment]
Apparently no, it doesn't import scale for IVCS bones.

**Comment by StoiaCode** [comment]
Did you read what I wrote? Make a scale with IVCS bones in Anamnesis, then import it into C+.

**Comment by seirralanzce** [comment]
Sorry let me rephrase, it does load the scale, but it doesn't load the option to play with in edit mode.
